semgrep/tests/e2e/snapshots/test_ci/test_github_ci_bad_base_sha/results.txt

```diff
@@ -0,0 +1,48 @@
+=== command
+CI="true" GITHUB_ACTIONS="true" GITHUB_EVENT_NAME="pull_request" GITHUB_REPOSITORY="project_name/project_name" GITHUB_RUN_ID="35" GITHUB_ACTOR="some_test_username" GITHUB_REF="some/branch-name" GITHUB_EVENT_PATH="<MASKED>" SEMGREP_APP_TOKEN="fake-key-from-tests" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep ci
+=== end of command
+
+=== exit code
+0
+=== end of exit code
+
+=== stdout - plain
+
+Findings:
+
+  bar.py 
+     eqeq-five
+        useless comparison to 5
+
+         ▶▶┆ Autofix ▶ bar == 2
+          1┆ bar == 2
+
+=== end of stdout - plain
+
+=== stderr - plain
+Scan environment:
+  versions    - semgrep <MASKED> on python <MASKED>
+  environment - running in environment github-actions, triggering event is pull_request
+  semgrep.dev - authenticated as org_name
+
+Fixing git state for github action pull request
+Not on head ref: 17b3114fb3d41437abc2a3e6499db84ad921e6f0; checking that out now.
```

Comment:
These commits should be stable based on how they were created so no need to mask them

Reply:
Using f7ee31 shows that we now use the correct merge-base. Older versions of semgrep would use 191a3ac